app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def index
+    @post = Post.all
+  end
+end
```

Comment:
- Good job done so far, but according to the project requirements, your controller should have a `show` action and a corresponding view file. Please fix this 🙏. Please note that it is okay for your controller to be empty 🙏